### PR TITLE
Export .env environment textures using WebP from Inspector

### PR DESCRIPTION
**Pull request by simonihmig**
Following up on #11102.

### CODE REVIEW

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx [hunk 1]
```diff
@@ -382,7 +393,16 @@ export class ToolsTabComponent extends PaneComponent {
                             <ButtonLineComponent label="Export to Babylon" onClick={() => this.exportBabylon()} />
                             {
                                 !scene.getEngine().premultipliedAlpha && scene.environmentTexture && scene.environmentTexture._prefiltered && scene.activeCamera &&
```
Comment (simonihmig) [inline inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:395]: I didn't want to touch this line (yet), but wonder why this `scene.environmentTexture._prefiltered` is needed?

With this still in place, I can e.g. drop a gltf that already has an .env environment into the sandbox, and the "Generate .env texture" show up. Without that check, I can easily export the .env without problems. In this case I think it does make sense, as it allows you to convert an existing PNG-based .env to the new WebP format!

TL;dr I'd like to change this to:

```suggestion
                                !scene.getEngine().premultipliedAlpha && scene.environmentTexture && scene.activeCamera &&
```

Any reason not to do so?

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx [hunk 2]
```diff
@@ -49,6 +54,7 @@ export class ToolsTabComponent extends PaneComponent {
     private _reflectorHostname: string = "localhost";
     private _reflectorPort: number = 1234;
     private _reflector: Reflector;
+    private _envOptions = { imageType: 0, imageQuality: 0.8 };
```
Comment (sebavan) [inline inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:57]: I guess imageType should be image/png by default :-)

Comment (sebavan) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:57]: Also is imageQuality 0.8 by default on png ? I thought the quality was depending on the mime type for the default value ?

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx [hunk 3]
```diff
@@ -251,7 +257,12 @@ export class ToolsTabComponent extends PaneComponent {
 
     createEnvTexture() {
         const scene = this.props.scene;
-        EnvironmentTextureTools.CreateEnvTextureAsync(scene.environmentTexture as CubeTexture)
+        EnvironmentTextureTools.CreateEnvTextureAsync(
+            scene.environmentTexture as CubeTexture,
+            {
+                imageType: envExportImageTypes[this._envOptions.imageType].imageType,
+                imageQuality: this._envOptions.imageQuality
```
Comment (sebavan) [inline inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:264]: Related to the first comment, the quality could be part of the envExportImageTypes array ?

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx (hunk 1 above)
Comment (sebavan) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:395]: We should not be able to generate a texture from a none prefiltered cubemap, basically if people are relying on a lightprobe and such and use real time filtering, it is not smthg we should convert in an env file.

**Review by sebavan — COMMENTED** [review]
Only a couple of nits :-) Thanks a lot

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx (hunk 2 above)
Comment (simonihmig) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:57]: > I guess imageType should be image/png by default :-)

It is. I used a numeric value for `imageType`, as it seems `<OptionsLineComponent>` can only deal with numbers as value, right? So this numbers refers to the index of ´envExportImageTypes´, and ´envExportImageTypes[0]´ is the entry for PNG!

> Also is imageQuality 0.8 by default on png

I believe PNG will ignore the quality all together, as it only supports lossless compression.

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx (hunk 3 above)
Comment (simonihmig) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:264]: I don't understand this... `envExportImageTypes` is a const/immutable array defining the options of `<OptionsLineComponent>`, and `imageQuality` is set by the user. So how should those be combined, or what exactly are you suggesting?

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx (hunk 2 above)
Comment (sebavan) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:57]: I see :-) it is just naming then, could you rename imageType in _envOptions to imageTypeIndex or something similar to prevent any confusion ? 

Basically looking into it too quickly I tought it was used as the string here.

File: inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx (hunk 3 above)
Comment (sebavan) [reply inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx:264]: Yup all good as pngs do not rely on it, and also coming from my previous confusion between the index and the type.

**Review by sebavan — CHANGES_REQUESTED** [review]
Basically mostly only one rename to go.

**Comment by sebavan** [comment]
@simonihmig I did the rename after the merge nw and thanks again for the contrib !!!!
